.github/workflows/release.yaml

```diff
@@ -17,7 +17,7 @@ jobs:
     steps:
       - uses: actions/checkout@v3
         with:
-          token: ${{ secrets.AUTO_PAT }}
+          token: ${{ secrets.AUTO_PAT_PROD }}
```

Comment:
i renamed this because canary build also uses `AUTO_PAT` naming and for a minute there i thought they were sharing the same secret. this makes it explicit, they're not sharing.